content/blog/2020-03-12-native-classroom/index.Rmarkdown

```diff
@@ -0,0 +1,93 @@
+---
+title: "Data Analysis for Social Justice in a Native Classroom"
+date: '2020-03-12'
+slug: native-classroom
+categories:
+  - teach
+author:
+  - "[Nic Thompson Gonzalez](http://www.nicolethompsongonzalez.com/)"
+photo:
+  URL: https://unsplash.com/photos/3KEJMfoHoMY
+  author: Graeme Nicholl
+---
+
+In this post, I'll share some insights from a work in progress. Last year, I began teaching an introduction to data science at the Southwestern Indian Polytechnic Institute [(SIPI)](https://www.sipi.edu/), a community college in Albuquerque, New Mexico with an exclusively Native American student-body. Although my professional background is research in behavioral ecology, my teaching passion is for data analysis. Why? Because I think data literacy and analytical expertise is one freely available tool to remedy many societal ills and injustices. My course datasets are therefore usually related to these ills. The material is currently delivered in an intensive 4-day workshop. One day, I hope it can be offered as a full course.
```

Comment:
Split para at "The material is currently delivered" and use that as intro to next para.